packages/node/src/lib/webhook-verify.ts

```diff
@@ -0,0 +1,25 @@
+const crypto = require('crypto');
+
+interface WebhookBody {
+  email: String;
+}
+
+export function verify(body: WebhookBody, signature: string, secret: string): WebhookBody {
+  const [fullMatch, time, readmeSignature] = /t\=(.*),v0\=(.*)/.exec(signature);
```

Comment:
<img width="925" alt="image" src="https://user-images.githubusercontent.com/848223/168782638-1e02e93c-6431-4042-802b-22701ddd699a.png">

Not sure what to make of this regex warning. Maybe try and anchor it to the start of the string? `^t\=(.*)` or only match it once `t\=(.*){1}`?